database/src/main/scala/cromwell/database/migration/restart/table/RenameWorkflowOptionKeysMigration.scala

```diff
@@ -0,0 +1,61 @@
+package cromwell.database.migration.restart.table
+
+import cromwell.database.migration.restart.table.RenameWorkflowOptionKeysMigration._
+import liquibase.database.jvm.JdbcConnection
+import spray.json._
+
+
+object RenameWorkflowOptionKeysMigration {
+  private val RenamedOptionKeys = Map(
+    "defaultRuntimeOptions" -> "default_runtime_attributes",
+    "workflowFailureMode" -> "workflow_failure_mode",
+    "workflow_log_dir" -> "final_workflow_log_dir",
+    "outputs_path" -> "final_workflow_outputs_dir",
+    "call_logs_dir" -> "final_call_logs_dir"
+  )
+
+  private val QueryWorkflowStore =
+    """
+       SELECT WORKFLOW_STORE_ID, WORKFLOW_OPTIONS FROM WORKFLOW_STORE
+    """
+
+  private val UpdateWorkflowStore =
+    """
+       UPDATE WORKFLOW_STORE SET WORKFLOW_OPTIONS = ? WHERE WORKFLOW_STORE_ID = ?
+    """
+}
+
+
+class RenameWorkflowOptionKeysMigration extends AbstractRestartMigration {
+
+  override protected def description: String = "Workflow option renaming"
+
+  override protected def doMigration(connection: JdbcConnection): Unit = {
+
+    def renameOptionKeys(field: JsField): JsField = {
+      field match {
+        case (oldName, value) if RenamedOptionKeys.contains(oldName) => RenamedOptionKeys(oldName) -> value
+        case noop => noop
+      }
+    }
+
+    val query = connection.createStatement()
+    lazy val insert = connection.prepareStatement(UpdateWorkflowStore)
+    query.execute(QueryWorkflowStore)
+    val results = query.getResultSet
+    while (results.next()) {
+      val options = results.getString(2)
+      if (!results.wasNull()) {
+        val optionsJson = options.parseJson
+        val newOptionsJson = optionsJson match {
+          case JsObject(fields) => JsObject(fields map renameOptionKeys)
+          case other => other // There really shouldn't be workflow options of other types, but if there are pass them through.
+        }
+
+        insert.setString(1, newOptionsJson.prettyPrint)
```

Comment:
ToL: Someday, if this isn't provenance, we'll switch to a structured, simpleton(?) storage. Today is not that day.